package.json

```diff
@@ -21,36 +21,35 @@
     "regenerator-runtime": "^0.13.11",
     "sbc-common-components": "3.0.8",
     "uuid": "^9.0.0",
-    "vue": "^2.7.14",
+    "vue": "3.3.4",
     "vue-router": "^3.6.5",
     "vue-the-mask": "^0.11.1",
-    "vue2-filters": "^0.14.0",
-    "vuelidate": "0.6.2",
-    "vuetify": "^2.6.15",
-    "vuex": "^3.6.2"
+    "vuelidate": "0.7.4",
+    "vuetify": "^3.1.8",
+    "vuex": "^4.0.0"
   },
   "devDependencies": {
     "@mdx-js/react": "^2.3.0",
-    "@storybook/addon-essentials": "^7.0.27",
-    "@storybook/builder-vite": "^7.0.27",
+    "@storybook/addon-essentials": "^7.5.3",
     "@storybook/mdx2-csf": "^1.1.0",
```

Comment:
Is this one still needed? Is this the latest version?

Reply:
Latest version but not needed

Reply:
And that's the latest version of addon-essentials? (Do we still need it?)

Reply:
Yes it is required and this is the latest version.